pkg/config/config.go

```diff
@@ -13,7 +13,11 @@ import (
 	"gopkg.in/yaml.v3"
 )
 
-const defaultMaxNumTracks = 4000
+const (
+	defaultLimitNumTracksPerCPU int32   = 400
+	defaultLimitMaxNumTracks    int32   = 8000
+	defaultLimitBytesPerSec     float32 = 5_000_000
```

Comment:
this is about 5 MB/sec ? that seems rather low as a limit (about 10 subs on a single publisher). I think we should leave it at 100MB/sec (slightly under 1Gbps)

Reply:
Good catch! I meant to set this limit at 5GB/sec.